Revit_Core_Engine/Convert/Environment/FromRevit/Duct.cs

```diff
@@ -58,7 +58,9 @@ public static List<BH.oM.MEP.System.Duct> DuctFromRevit(this Autodesk.Revit.DB.M
                 bhomDucts = new List<BH.oM.MEP.System.Duct>();
             }
 
-            List<BH.oM.Geometry.Line> queried = Query.LocationCurveMEP(revitDuct, settings);
+            bool isStartConnected = false;
+            bool isEndConnected = false;
+            List<BH.oM.Geometry.Line> queried = Query.LocationCurveMEP(revitDuct, out isStartConnected, out isEndConnected, settings);
```

Comment:
Will `isStartConnected` and `isEndConnected` be relevant for this convert at any point? If not, I would consider adding an overload without `out` parameters that would call the one with them and simply not return them.

This comment refers to other converts as well.